frontend/src/theme.scss

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
@jculvey I think that we might want to remove this file as well, right?

(if so, do you prefer me to do it now or in a followup PR?)

Reply:
Yeah, if we're not using any of that stuff anymore go for it. Better to do it now I think to make sure we've covered our bases.

Reply:
no, it is being used, that's why I kept it

so I can either remove it and remove the scss code using its sass variables, or I can just remove it and replace usages of its sass variables with the hardcoded values

I think it's mostly used for the colors, I guess I can try to completely remove it alongside the variables usages and check that it doesn't break the ui too much, if it does I can hardcode the values or try to apply some tailwind classes

Reply:
I did a bit of both (removed things that didn't require a complex refactoring and replaced variables with hardcoded values everywhere else)